arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "How to customize your Orbit chain's precompiles"
+sidebar_label: "Customize your chain's precompiles"
+description: "Learn how (and when) to customize your Orbit chain's precompiles"
+author: jasonwan
+sidebar_position: 2
+---
+
+import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+import UnderConstructionPartial from '../../partials/_under-construction-banner-partial.md';
+
+<UnderConstructionPartial />
+
+There are two ways to customize your chain's precompiles:
+
+1.  Add new methods to to an existing [precompile](https://github.com/OffchainLabs/nitro-contracts/tree/97cfbe00ff0eea4d7f5f5f3afb01598c19ddabc4/src/precompiles).
+2.  Create a new precompile.
+
+### Prerequisites
+
+Clone the Nitro repository before you begin:
+
+```shell
+git clone <https://github.com/OffchainLabs/nitro.git>
```

Comment:
Since this is related to Orbit chains, it might be good to download the latest orbit-ready nitro branch/tag. The parameter for doing that is --branch, or -b, iirc.